transformer_engine/common/include/transformer_engine/softmax.h

```diff
@@ -0,0 +1,82 @@
+/*************************************************************************
+ * Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_SOFTMAX_H_
+#define TRANSFORMER_ENGINE_SOFTMAX_H_
+
+#include <cuda_fp16.h>
+#include <cuda_bf16.h>
+#include "transformer_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+inline int log2_ceil(int value) {
```

Comment:
This is an actual API header that exposes library functions. This random function should not be here. Please move it to some file like utils or common that is internal to TE.